src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -1160,5 +1180,29 @@ public RunningBackup OnGoingBackup(long taskId)
                 IsFull = periodicBackup.BackupStatus.IsFull
             };
         }
+
+        public IReadOnlyDictionary<string, long> GetLastProcessedDocumentTombstonesPerCollection()
+        {
+            var results = new Dictionary<string,long>();
+
+            foreach (var periodicBackup in _periodicBackups.Values)
+            {
+                foreach (var etagItem in periodicBackup.BackupStatus.LastTombstoneEtagsByCollection)
```

Comment:
Why are you tracking just tombstones here? Why not just track the last etag that you backup, globally?
This is something that is actually already going on, and then we can just output that value for each of the collections that exists.